Application/Cache/NameDeletedCache.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Core.Cache;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.Cache
+{
+    public class NameDeletedCache :InMemoryCache, INameDeletedCache
```

Comment:
You can safely remove this cache. We won't be needing it.

Reply:
Done